tests/integration_tests/configs/upgrade-test-package.nix

```diff
@@ -1,9 +1,9 @@
 let
   pkgs = import ../../../nix { };
-  fetchEthermint = rev: builtins.fetchTarball "https://github.com/evmos/ethermint/archive/${rev}.tar.gz";
+  fetchEthermint = rev: builtins.fetchTarball "https://github.com/mmsqe/ethermint/archive/${rev}.tar.gz";
   released = pkgs.buildGo118Module rec {
     name = "ethermintd";
-    src = fetchEthermint "d29cdad6e667f6089dfecbedd36bb8d3a2a7d025";
+    src = fetchEthermint "0e4d41eef5f7983f84206b1bb07e88ed3a9cd44b";
```

Comment:
not sure if can backport [1392](https://github.com/evmos/ethermint/pull/1392/files) and [1405](https://github.com/evmos/ethermint/pull/1405/files) to v0.19.x

Reply:
keep migrate related stuff no change but only test with patch old binary